paramiko/config.py

```diff
@@ -295,3 +295,29 @@ def __str__(self):
             # Cache
             self.fqdn = fqdn
         return self.fqdn
+
+
+class SSHConfigDict(dict):
+    """A dictionary wrapper for ssh host configurations.
+
+    This class introduces some usage niceties for consumers of SSHConfig,
+    specifically around the issue of variable type conversions. This offers
+    as_bool(key) and as_int(key) for the current raw string values in
+    SSHConfig"""
+
+    def __init__(self, *args, **kwargs):
```

Comment:
Do we still need this now that we're subclassing just dict and not UserDict?

Reply:
Good question. Probably not, but it doesn't hurt.